src/main/java/interface_adapter/signup/student_signup/StudentSignupPresenter.java

```diff
@@ -27,7 +27,7 @@ public StudentSignupPresenter(ViewManagerModel viewManagerModel,
     public void prepareSuccessView(StudentSignupOutputData response) {
         // On success, switch to the login view.
         final LoginState loginState = loginViewModel.getState();
-        loginState.setIdentifier(response.getUsername());
+        loginState.setIdentifier(response.getEmail());
```

Comment:
The deleted line's purpose was to put the Student's username into the login after signing up, for convenience. Just confirming, are we changing it so all users login with their emails?

Reply:
Okay thanks for the clarification!

Reply:
👍